workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/SpatialOSBehaviourLibrary.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
```

Comment:
Should be `Behavior`, we use US spellings. 
cc @ElleEss

Reply:
The name derives from Unity's MonoBehaviours though, so an exception seems sensible in this case?

Reply:
I think behaviour is the one English spelling that we use, because MonoBehaviour uses it :P
If not in docs it would probably make sense to keep this consistent in code.